src/js/utils/background.js

```diff
@@ -117,8 +117,12 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
       return css`
         background: ${backgroundColor};
         color: ${normalizeColor(
-          textColor[colorIsDark(backgroundColor) ? 'dark' : 'light'] ||
-            textColor,
+          textColor[
+            colorIsDark(backgroundColor) ||
```

Comment:
Rather than calling `colorIsDark` twice, how about saving the result in a `const` after line 116?

Reply:
good call.